ci-operator/step-registry/rosa/sts/cluster/create/rosa-sts-cluster-create-commands.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+CLUSTER_NAME=${CLUSTER_NAME:-$NAMESPACE}
+ACCOUNT_ROLES_PREFIX=${ACCOUNT_ROLES_PREFIX:-"ManagedOpenshift"}
+COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-"m5.xlarge"}
+CLOUD_PROVIDER_REGION=${CLOUD_PROVIDER_REGION:-"us-east-2"}
```

Comment:
should be the same as `$LEASE_RESOURCE`?

Reply:
yes.  according to @jianlinliu 's comment, will update it to `${LEASED_RESOURCE}`